src/workloads/scale/MixedWorkloadsGenny.yml

```diff
@@ -33,6 +33,70 @@ Clients:
     QueryOptions:
       maxPoolSize: 500
 
+ActorTemplates:
+- TemplateName: UpdateTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "Update"}}
+    Type: CrudActor
+    Database: *dbname
+    ClientName: Update
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyIn:
+        Active: [{^Parameter: {Name: "Phase", Default: 1}}]
+        Max: 10
+        Config:
+          ExternalPhaseConfig:
```

Comment:
hrm. Do we still really want/need `ExternalPhaseConfig` **and** templates? It's kinda neat that it works and that you can `genny evaluate` to undo all of the magic, but it seems really hard to reason about. I would almost be inclined to say you can't use `ExternalPhaseConfig` inside `ActorTemplate` but perhaps that's overly limiting?

Reply:
My gut instinct is that more composability is usually a good thing, and as per the slack convo request to allow the `ActorTemplate`s to someday be a separate file that can be specified, I think that sort of composability to mix/match actor templates and phases may be desired by users over adding an artificial limit.

Right now all this preprocessing is just parameterized substitutions with a couple different convenience syntaxes, so I feel like if we keep to that principle we should hopefully be able to manage the complexity?

Reply:
I'm afraid of somebody looking at a complicated example involving both templates and external phases and coming to the conclusion that genny is too hard to configure and use.

So ya such a limit would be artificial, but I view it more as "guardrails" encouraging somebody to simplify and to maybe come to us if they want the rails removed.

I'm really fine to leave it--up to you--but perhaps spit out a warning or make it clear in the docs that using templates plus external phases might mean writing a new Actor would be simpler. And let's also mention in the docs/examples of templates and external phases that `genny evaluate` can be used to show the yaml with all includes/parameters applied.